app/views/users/index.html.erb

```diff
@@ -1,8 +1,7 @@
 <h1><%= t(".title") %></h1>
 <ul>
   <% @users.each do |user| %>
-    <li><%= user.name %></li>
-    <li><%= user.email %></li>
+    <%= render "detail", user: user %>
     </br>
```

Comment:
<br> is a presentation tag, which is not needed because the same effect can be achieved using css styles.

Reply:
Done sha: 78a6e82